src/clj_jgit/porcelain.clj

```diff
@@ -415,11 +415,12 @@
   "
   ([^Git repo]
```

Comment:
New optional args should also be passed here.

Reply:
Unfortunately, it is not possible to add the optional args here because it would result in a conflict between the two arities, both of which can take accept 2 arguments:

```
Exception in thread "main" java.lang.RuntimeException: Can't have more than 1 variadic overload, compiling:(clj_jgit/porcelain.clj:410:1)
```

Effectively, this means you have to include an `"origin"` argument if you want to push tags. I realize this is not ideal.

As an alternative, I could leave the existing `git-push` the way it is, and add a new `git-push-tags` function that has the same arities as `git-push`... what do you think?

Reply:
Hmm or we could drop the `git-push` overload and just add a variadic `:remote` keyword that defaults to `"origin"`. Probably better anyways as more options for `git-push` might be needed in the future. In theory a breaking change but probably won't affect anyone in practice.

Reply:
I like that idea. :+1: